File: fix-id.ts
import * as fs from 'fs';
const file = 'src/components/screens/CertificationTrackingScreen.tsx';
let content = fs.readFileSync(file, 'utf8');

content = content.replace(/activeCompany\?\.id/g, "activeCompany?.companyId");
content = content.replace(/activeCompany\.id/g, "activeCompany.companyId");

fs.writeFileSync(file, content);

File: fix_id.py
with open('src/components/screens/IdentityBadgeScreen.tsx', 'r') as f:
    content = f.read()

content = content.replace("""                      <button 
                        onClick={() => setVerifyInput('IDB-DEMO-123')}
                        className="text-xs text-indigo-600 hover:underline text-center col-span-2"
                      >
                        Try Demo QR Identifier
                      </button>""", "")

with open('src/components/screens/IdentityBadgeScreen.tsx', 'w') as f:
    f.write(content)
